src/init/wazuh-client.sh

```diff
@@ -94,7 +95,7 @@ unlock()
 help()
 {
     # Help message
-    echo "Usage: $0 {start|stop|restart|status}";
+    echo "Usage: $0 {start|stop|restart|status|info [-v -r -d -t]}";
```

Comment:
Shouldn't we remove the `-d` switch here?

Reply:
You are right. Fixed.